inc/components/components/image/component.json

```diff
@@ -28,35 +28,46 @@
       "description": "Optional. URL for the image source if \"src\" is not set.",
       "type": "string"
     },
+    "id": {
+      "description": "Optional. An attachment ID for an image. Will be overridden if \"src\" is set.",
+      "type": "int",
+      "hidden": true
+    },
     "loading": {
-      "defaul": "lazy",
-      "description": "Required. \"eager\" or \"lazy\".",
+      "default": "lazy",
+      "description": "Lazy loading setting. Either \"eager\" or \"lazy\".",
       "type": "string"
     },
     "object_fit": {
       "default": "cover",
       "description": "Object fit property.",
       "type": "string"
     },
-    "picture_sources": {
-      "description": "Optional. Array of sources for the <picture> tag.",
-      "type": "array"
-    },
     "show_meta": {
       "default": true,
       "description": "Display meta.",
       "type": "bool"
     },
+    "size": {
+      "default": "full",
+      "description": "A WordPress image size to use. Default 'full'.",
+      "type": "string",
+      "hidden": true
+    },
     "src": {
       "description": "Optional. URL for the image source.",
       "type": "string"
     },
+    "srcset": {
+      "description":"A source list string for delivering responsive images.",
+      "type": "string"
+    },
     "style": {
       "default": [],
       "type": "array"
     },
-    "photon_transformations": {
-      "description": "A set of Photon API transformations in the order they should be performed",
+    "query_args": {
```

Comment:
We can use this more generically for photon args or for other DAM/CDN arguments, when necessary.